test/api_get_product_test.dart

```diff
@@ -1430,4 +1432,81 @@ void main() {
       expect(productResult.product!.dataQualityWarningsTags, isNull);
     });
   });
+
+  test('check nutrients modifier (set)', () async {
```

Comment:
Can't understand the title, more specifically: "(set)".

Reply:
I forgot to include a few changes.
Indeed, it wasn't understandable